comp/dogstatsd/packets/telemetry.go

```diff
@@ -15,7 +15,11 @@ var (
 	// though all buffers currently share a single channel it's still worth
 	// tagging it with listener_id in case this changes later.
 	tlmChannelSize = telemetry.NewGauge("dogstatsd", "packets_channel_size",
+		[]string{}, "Number of packets in the packets channel")
+	tlmChannelSizePackets = telemetry.NewGauge("dogstatsd", "packets_channel_size_packets",
```

Comment:
Can we please give a more descriptive name for this metric? Otherwise it isn't clear how it is different from `packets_channel_size`. Something like:

```suggestion
	tlmChannelSizePackets = telemetry.NewGauge("dogstatsd", "packets_channel_size_by_listener",
```